### PR TITLE
Update remote chapter for new remote, fixes #493

### LINKED ISSUE #493: Some changes are needed to the remote app pages
Remote Control:
1. The URLs which point to our apps on the apple app store are currently broken.
2. The blue 'info bar' no longer correctly reflects how the remote app works as of v17. It should now be backwards compatible with two prior versions of Titan.

Operating the Remote:
1. The final sentence, "You cannot record cues or groups using the remote.", is incorrect. You can record both of these in the remote now.

### PR DESCRIPTION
**Pull request by congoblue**
Updated images and text for new remote.

### CODE REVIEW

File: docs/remote-control/operating-the-remote.md [hunk 1]
```diff
@@ -94,13 +108,17 @@ an active playback.
 
 
 
-## Recording Palettes
+## Recording Palettes, Groups and Cues
```
Comment (PeterBuddAvo) [inline docs/remote-control/operating-the-remote.md:111]: Somewhere in this section it should probably be mentioned that when recording playbacks the consoles record mode is followed.

File: docs/remote-control/operating-the-remote.md [hunk 2]
```diff
@@ -53,23 +68,22 @@ button, for example:
...
 
-![Fixture Control in Titan Remote Android App](/docs/images/Fixture-Control-in-Titan-Remote-Android-App.png)
+Use the **IPCGBES** buttons at the top to select the group of attributes you want
+to control, and scroll wheels will be shown for the available functions.
+
+![Colour Control in Titan Remote Android App](/docs/images/Remote-Colour-Attributes.png)
 
 The wheels show function values (such as colours or gobo names). To go
-to the next function, just touch the arrows at the top or bottom of the
+to the next function if the attribute has ranges, just touch the arrows at the top or bottom of the
```
Comment (PeterBuddAvo) [inline docs/remote-control/operating-the-remote.md:86]: If this is the only section on the wheels we should probably mention that it is possible to pinch to fan an attribute.

We should probably mention that for function ranges you need to tap up/down.

File: docs/remote-control/operating-the-remote.md [hunk 3]
```diff
@@ -6,29 +6,44 @@ sidebar_label: Operating the Remote
...
-the remote when no real console is connected.
+the remote when no real console is connected. If your console does not show
+up in the list, you can enter its IP address manually by clicking the + button 
+in the top right hand corner of the screen.
+
+![Titan Remote Android App with wheels](/docs/images/Remote-Colour-Attributes.png)
 
-![Titan Remote Android App with keypad](/docs/images/Titan-Remote-Android-App-Control-Selection.png)
+The app title bar should be green showing good connection. If the title bar turns
+red then you do not have connection to the console.
```
Comment (PeterBuddAvo) [inline docs/remote-control/operating-the-remote.md:25]: The title bar now has an orange state which it shows up when the connection to the console is slow. We have deemed slow as greater than approximately 250ms to respond to a message.

**Review by PeterBuddAvo — COMMENTED** [review]
I have noted a few specific changes below. I have also attached a document written internally that dealt with some internal feedback.

I wonder if it is worth a review of the connection page https://manual.avolites.com/docs/remote-control/setting-up-the-remote. It either contains too little or too much information. It might have been enough just to say that you need wireless network set up, and we recommend it is private and non internet connected. If we are leaving in everything about having an access point we should probably provide some guidance as to it's physical location. This could include being line of sight with no obstructions.
[v17 Titan Remote App FAQ.docx](https://github.com/user-attachments/files/17632765/v17.Titan.Remote.App.FAQ.docx)

File: docs/remote-control/operating-the-remote.md [hunk 4]
```diff
@@ -6,29 +6,41 @@ sidebar_label: Operating the Remote
...
-![Titan Remote Android App with keypad](/docs/images/Titan-Remote-Android-App-Control-Selection.png)
+![Titan Remote Android App status bar](/docs/images/Remote-Topbar.png)
 
-Choose what you want to control using the row of buttons across the
-middle of the app. The Keypad button enables or disables the numeric pad
-(the keypad overlays all the other screens, so you will need to turn it
+The app title bar shows the network connection status: 
+- green: good connection. The response time from the
+console is shown on the top right. 
+- red: you have lost connection to the console. Move closer or reposition your WiFi point.
```
Comment (PeterBuddAvo) [inline docs/remote-control/operating-the-remote.md:18]: If the connection bar is red or orange you probably need to look at the network setup. Maybe just write underneath. 

If the title bar is orange or red you should try moving closer to or repositioning the access point.

**Review by PeterBuddAvo — APPROVED** [review]
I made a minor comment but otherwise it's fine with me. I think Olie wanted to look at these changes, and I asked Joe if he wanted to have a look.

File: docs/remote-control.md [hunk 5]
```diff
@@ -6,24 +6,24 @@ sidebar_label: Remote Control
 
 import Keys from '@site/src/components/key.ts';
 
-With the addition of an external Wireless Access Point, the console can
-be remote controlled using **WiFi** from an [Android](https://play.google.com/store/apps/developer?id=Avolites+Ltd) or [iOS](https://apps.apple.com/us/developer/avolites-ltd/id688791177#see-all/i-phone-apps) mobile device
```
Comment (joe-misra) [inline docs/remote-control.md:10]: The IOS link doesn't seem to work for me. Removing the ending so it's just https://apps.apple.com/us/developer/avolites-ltd/id688791177 made it work.

File: docs/remote-control/operating-the-remote.md (hunk 4 above)
Comment (joe-misra) [reply docs/remote-control/operating-the-remote.md:18]: Worth noting that red doesn't necessarily mean it's lost connection, only that the ping is greater than 1000ms

**Pull request merged**